bindings/ios/MEGASdk.h

```diff
@@ -5400,6 +5401,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
  */
 - (void)killSession:(uint64_t)sessionHandle;
 
+/**
+ * @brief Returns the deadline to remedy the storage overquota situation
+ *
+ * This value is valid only when [MEGASdk getUserData] has been called after
+ * receiving a callback [MEGAGlobalDelegate onEvent:event] of type
+ * EventStorage, reporting StorageStatePaywall.
+ * The value will become invalid once the state of storage changes.
+ *
+ * @return `NSDate` instance representing the deadline to remedy the overquota
+*/
+- (nonnull NSDate *)overquotaDeadlineDate;
```

Comment:
I think we can remove `nonnull` here. We have `NS_ASSUME_NONNULL_BEGIN` at the beginning of the file

Reply:
yup, nonnull by default